cmd/sonarExecuteScan.go

```diff
@@ -215,14 +238,14 @@ func runSonar(config sonarExecuteScanOptions, client piperhttp.Downloader, runne
 		log.Entry().Warn("no measurements are fetched due to missing credentials")
 		return nil
 	}
-	taskService := SonarUtils.NewTaskService(taskReport.ServerURL, config.Token, taskReport.TaskID, apiClient)
+	taskService := SonarUtils.NewTaskService(config.ServerURL, config.Token, taskReport.TaskID, apiClient)
```

Comment:
When the server url is provided via Jenkins Sonar plugin (env var) this will point to the wrong url.

Reply:
But, when the proxy is enabled, the report contains the actual address of the instance but not the virtual one. To which url does it point to when set it via sonar plugin?